client/app/bundles/course/group/pages/GroupShow/GroupTableCard.jsx

```diff
@@ -85,9 +131,23 @@ const GroupTableCard = ({ group }) => {
           {members.map((m, index) => (
             <TableRow key={m.id} style={styles.rowHeight}>
               <TableCell style={styles.rowHeight}>{index + 1}</TableCell>
-              <TableCell style={styles.rowHeight}>{m.name}</TableCell>
               <TableCell style={styles.rowHeight}>
-                {roles[m.groupRole]}
+                <div className="flex grow">
+                  {m.name}
+                  {m.isPhantom ? <GhostIcon /> : ''}
+                </div>
+              </TableCell>
+              <TableCell style={styles.rowHeight}>
+                <Chip
```

Comment:
I think this is reused in `GroupManagerList` right? You can abstract this into a reusable component

Reply:
need more advise regarding this. tried to abstract this into reusable component, but seems like the value `user` is not passed over to the function

Reply:
done